src/styles/components/typeahead.less

```diff
@@ -0,0 +1,27 @@
+.typeahead.inverse {
```

Comment:
hmm it seemed convenient to be able to to make the entire typeahead inverse or non-inverse styled with just one css class